app/lib/steps/helper.rb

```diff
@@ -53,8 +53,7 @@ def step_groups_for(session_data)
       end
 
       def all_sections(session_data = nil)
-        initial_sections = [CaseDetailsSection,
-                            ApplicantDetailsSection,
+        initial_sections = [NonFinanceSection,
```

Comment:
just suggesting that NonFinancial might be better if we are going to formalise using it, but thats just the grammar police in me